### ISSUE TITLE
Request: Import/Export Multiple snippets from different groups...retain groups

### ISSUE THREAD

**Issue by mi6300m** [xmichelo/Beeftext#130]
An issue I have is that if I select multiple snippets from various groups, and export those to a work colleague, on their end it only allows them to sort into one folder and then they have to manually sort them out one by one. 

Is there a way to retain, or create missing groups if imported to another user?

**Comment by xmichelo** [comment]
No. It is not possible at the moment. It is of course an interesting feature.
